sbtc/src/rpc.rs

```diff
@@ -117,12 +116,13 @@ impl BtcClient {
     ///
     /// Modified from the bitcoin-core docs[1]:
     ///
-    /// This returns a "conservative" estimate, which is potentially
-    /// returns a higher fee rate and is more likely to be sufficient for
-    /// the desired target, but is not as responsive to short term drops in
-    /// the prevailing fee market. Also, the docs mention the response is
-    /// in BTC/kB, but from the comments in  bitcoin-core[2] this is really
-    /// BTC/kvB (kvB is kilo-vbyte).
+    /// Bitcoin-core has too different modes for fee rate estimation,
```

Comment:
```suggestion
    /// Bitcoin-core has two different modes for fee rate estimation,
```

Reply:
Oh oops, good find. Will update.

Reply:
Done in https://github.com/stacks-network/sbtc/pull/385/commits/a2d8c5dec94c161d1b2448d4f82e407d272be2e5.